### PR TITLE
Added ratings to html discussion details

### PR DESCRIPTION
**Pull request by vanwagonet**
refs: MBL-14161
affects: Student, Teacher
release note: none

### CODE REVIEW

**Comment by inst-danger** [comment]
## Latest Installable Builds

(will redirect to this page if no builds exist yet)
student: [Link](https://rebrand.ly/skvempz) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Fskvempz)
teacher: [Link](https://rebrand.ly/tcjzjwq) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Ftcjzjwq)
parent: [Link](https://rebrand.ly/fsorlev) | [QR](https://qr.rebrandly.com/v1/qrcode?shortUrl=rebrand%2Ely%2Ffsorlev)

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


#### Affected Apps: Student, Teacher

[MBL-14161](https://instructure.atlassian.net/browse/MBL-14161)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.68% | 93.68% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Core/Core/Discussions/DiscussionDetailsViewController.swift [hunk 1]
```diff
@@ -448,7 +485,7 @@ extension DiscussionDetailsViewController {
             text-align: center;
             text-decoration: none;
         ">
-            \(CoreWebView.htmlString(NSLocalizedString("View more replies", bundle: .core, comment: "")))
+            \(t(NSLocalizedString("View more replies", bundle: .core, comment: "")))
```
Comment (bluwave) [inline Core/Core/Discussions/DiscussionDetailsViewController.swift:488]: What does this `t` do?

Comment (bluwave) [reply Core/Core/Discussions/DiscussionDetailsViewController.swift:488]: Oh...it's a method. I was scanning from the bottom up 😂

Comment (vanwagonet) [reply Core/Core/Discussions/DiscussionDetailsViewController.swift:488]: yeah I was trying to trim up the really long common escaping.

**Pull request merged**